PULL_REQUEST_TEMPLATE.md

```diff
@@ -13,70 +13,77 @@
   Why are you making this change? There is nothing more important
   to provide to the reviewer and to future readers than the cause
   that gave rise to this pull request. Be careful to avoid circular
-  statements like "the purpose is to update the schema." and
-  instead provide an explanation like "there is more data to be provided and stored for Purchase Orders 
+  statements like "the purpose is to update the schema." 
+  instead, provide an explanation like "there is more data to be provided and stored for Purchase Orders 
   which is currently missing in the schema"
 
   The purpose may seem self-evident to you now, but the standard to
   hold yourself to should be "can a developer parachuting into this
-  project reconstruct the necessary context merely by reading this
+  the project reconstructs the necessary context merely by reading this
   section."
 
   If you have a relevant JIRA issue, add a link directly to the issue URL here.
-  Example: https://issues.folio.org/browse/MODORDERS-70
+  Example: https://issues.folio.org/browse/MODINVOSTO-70
  -->
 
 ## Approach
 <!--
  How does this change fulfill the purpose? It's best to talk
  high-level strategy and avoid code-splaining the commit history.
 
- The goal is not only to explain what you did, but help other
+ The goal is not only to explain what you did but help other
  developers *work* with your solution in the future.
 -->
 
 #### TODOS and Open Questions
 <!-- OPTIONAL
 - [ ] Use GitHub checklists. When solved, check the box and explain the answer.
+- [ ] Check logging.
 -->
 
 ## Learning
 <!-- OPTIONAL
-  Help out not only your reviewer, but also your fellow developer!
+  Help out not only your reviewer but also your fellow developer!
   Sometimes there are key pieces of information that you used to come up
   with your solution. Don't let all that hard work go to waste! A
   pull request is a *perfect opportunity to share the learning that
-  you did. Add links to blog posts, patterns, libraries or addons used
+  you did. Add links to blog posts, patterns, libraries, or add-ons used
   to solve this problem.
 -->
 
 ## Pre-Merge Checklist:
-Before merging this PR, please go through the following list and take appropriate actions.
+Before merging this PR, please go through the following list and take appropriate action.
```

Comment:
Please put it back as it was on the deleted line

Reply:
ok, I have fixed